config/locales/en.yml

```diff
@@ -10,3 +10,8 @@ en:
       invalid_content_type: Invalid content type header
       missing_param: A required param is missing
       not_found: Couldn't find the record
+  email:
+    ticket:
+      new_ticket: Your ticket %{} was created
+    user:
+      you_do_not_belong_here: You don't belong here. Please, create a new account
```

Comment:
Why were the words `You don't belong here` chosen? It sounds kind of weird to me in this context.

Reply:
is to be sarcastic and realize that you have to change the text.  This is an internal project, to learn Action Mailbox @AgostinaDev